news/_posts/2022-01-18-contributor-interview-krystian-podemski.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "From contributor to maintainer: Krystian Podemski's experience with the PrestaShop project"
+subtitle: "Discover the faces behind the commits"
+date:   2022-01-18 
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2021/01/krystian-podemski.jpg
```

Comment:
```suggestion
image: /assets/images/2022/01/krystian-podemski.jpg
```
I suppose ^^

Reply:
Second time in a week... guess I'm stuck in 2021. Not the best year to be stuck though 😅